### DISCUSSION TITLE
Writing workflow - moving multiple cards quickly font size

### DISCUSSION THREAD

**Discussion by William100-0** [mgmeyers/obsidian-kanban#508] [Ideas & Feature Requests]
Using the plugin for my workflow at the moment to drag and drop atomic notes into an outline on other kanban boards. Finding it very, very helpful, but a bit slow at times (I do have hundreds/thousands of these notes in kanban boards). At the moment I plan to move my notes around to increasingly smaller kanban boards but

Does anyone know if 

a) if it is possible to move more than one note around at a time (or will be in the future) and

b) to change the font size on the cards? (quite small at the moment)

**Comment by mgmeyers** [comment]
Currently, the easiest way to move several cards at once is to edit the board as markdown where you can cut and paste lines where you want them. I'll support dragging multiple cards at some point in the future, but it'll be a while before I get there.

Currently you'll have to use a [css snippet](https://help.obsidian.md/How+to/Add+custom+styles#Use+Themes+and+or+CSS+snippets) to change the fonts. Here's an example:

```
/* Change the font-size of card text */
.kanban-plugin__grow-wrap > textarea, 
.kanban-plugin__grow-wrap:after,
.kanban-plugin__markdown-preview-view {
  font-size: 16px !important;
}

/* Change the font-size of card metadata */
.kanban-plugin__item-metadata:not(:empty) {
  font-size: 14px !important;
}
```

**Reply by William100-0** [reply]
Understood - thanks very much! (and once again - it is a great plugin, absolutely invaluable!)

**Comment by William100-0** [comment]
Do you happen to have an update on when moving multiple cards might be possible? It is a feature I absolutely with Speare - makes moving hundreds of cards round as part of a writing project much easier. It would be great to have it with this plugin. Thanks for all your work on it
